providers/efa/verbs.c

```diff
@@ -1102,6 +1102,23 @@ int efa_post_recv(struct ibv_qp *ibvqp, struct ibv_recv_wr *wr,
 	return err;
 }
 
+int efadv_query_ah(struct ibv_ah *ibvah, struct efadv_ah_attr *attr)
+{
+	uint64_t comp_mask_out = 0;
+
+	if (!is_efa_dev(ibvah->context->device))
+		return EOPNOTSUPP;
+
+	attr->version = 0;
+	attr->flags = 0;
+
```

Comment:
No idea why you are putting version and flag 0's here. If the current caller can't do anything with this then just drop it and use comp_mask if someday something is needed. We shouldn't have multiple layers of future proofing.

Reply:
Done